modules/Bio/EnsEMBL/Compara/PipeConfig/UpdateReferenceDatabase_conf.pm

```diff
@@ -62,11 +62,10 @@ sub default_options {
         'pipeline_name' => 'update_references_' . $self->o('rel_with_suffix'),
         'backups_dir'   => $self->o('pipeline_dir') . '/reference_db_backups/',
 
-        # shared location to symlink to fastas for orthofinder
+        # shared locations to symlink and copy fastas for orthofinder
         'shared_fasta_dir' => $self->o('shared_hps_dir') . '/reference_fasta_symlinks/',
-
-        # orthofinder executable
-        # 'orthofinder_exe' => $self->o('orthofinder_exe'),
+        'ssh_ip_loc' => '45.88.81.155',
```

Comment:
is this because of the mock bucket? or is i the intention to have a hard coded ip in this?

Reply:
The intention is to have the ip here hardcoded - it is not expected to chance